burn-image-osx.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
I would prefer to have a `burn-image.sh` that checks whether the platform is OS X, so we can add support for different platforms later (i.e. provide an alternative to the `diskutil` commands)